### PR TITLE
7.x

### PR DESCRIPTION
**Pull request by pc37utn**
**JIRA Ticket**: (link)

Updating readme
# What does this Pull Request do?

   Changes a screenshot and updates the readme
# What's new?

 Corrected some text in the description and changed the screen shot to a newer one
# How should this be tested?

 Compare to documentation wiki.
# Interested parties

Tag @DonRichards @jordandukart

### CODE REVIEW

**Comment by DonRichards** [comment]
Ping @jordandukart

**Comment by pc37utn** [comment]
corrected those errors, thanks!

**Comment by DiegoPino** [comment]
@pc37utn thanks 👍

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Introduction
 
-This module provides an implementation the Open Archives Initiative Protocol for Metadata Harvesting (OAI-PMH) for Islandora fedora repository with a solr index. By implementing the islanodra_oa module, you can expose content (its metadata) as an OAI-PMH repository. It will then be accessible by OAI harvesters For further OAI documentation. For more info, please see [this]( http://www.openarchives.org/OAI/openarchivesprotocol.html) link.
+This module provides an implementation of a data provider according to the Open Archives Initiative Protocol for Metadata Harvesting (OAI-PMH) for an Islandora fedora repository with a solr index. By implementing the islandora_oai module, you can expose content (its metadata) as an OAI-PMH repository. It will then be accessible by OAI harvesters. For further OAI documentation. For more info, please see [this]( http://www.openarchives.org/OAI/openarchivesprotocol.html) link.
```
Comment (jordandukart) [inline README.md]: Capital F for Fedora?

**Comment by jordandukart** [comment]
Looks good to me

**Pull request merged**